docs/refdata_gen.ipynb

```diff
@@ -0,0 +1,197 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Generate TARDIS Reference Data"
```

Comment:
I would change the title (and also the name of the notebook). The notebook does not generate what we usually refer to as reference data for TARDIS.

Reply:
Renamed the file in https://github.com/tardis-sn/carsus/pull/313/commits/57be60970fcc59718ad67e691701cbea23e37163, but I'm still not sure if that is the appropriate name. Do you have any other suggestions?

Reply:
The new name looks good to me!